net/can/can_sendmsg.c

```diff
@@ -106,11 +106,13 @@ static uint16_t psock_send_eventhandler(FAR struct net_driver_s *dev,
         {
           /* Copy the packet data into the device packet buffer and send it */
 
-          devif_send(dev, pstate->snd_buffer, pstate->snd_buflen, 0);
+          int ret = devif_send(dev, pstate->snd_buffer,
+                               pstate->snd_buflen, 0);
           dev->d_len = dev->d_sndlen;
```

Comment:
should we move this after line 117 or remove it?

Reply:
It was like this before, and this is only for CAN.
I don't know if this assignment is important for the CAN system or its drivers, and I cannot test it.

I would prefer to leave it as is, and not touch it. Unless you are sure and insist.